### ISSUE TITLE
FrustumMesh Bug?

### ISSUE THREAD

**Issue by metelmann** [FXyz/FXyz#22]
The following example does render a cone from 0,-10,0 to 0,-20,0
```
kegel = new FrustumMesh(); // majorrad, minrad, height, level
kegel.setAxisEnd(new Point3D(0f, 10.0f, 0f));
kegel.setAxisOrigin(new Point3D(0f, -10.0f, 0f));
kegel.setMajorRadius(.5);
kegel.setTextureModeNone(new Color(0, 1, 0, 1));
group.getChildren().add(kegel);
```

it works correctly (except for direction) once you change origin and end, i.e.
```
kegel.setAxisEnd(new Point3D(0f, -10.0f, 0f));
kegel.setAxisOrigin(new Point3D(0f, 10.0f, 0f));
```

or if the x values are not the same, so the following does give the (nearly) correct cone:
```
kegel.setAxisEnd(new Point3D(0f, 10.0f, 0f));
kegel.setAxisOrigin(new Point3D(0.0001f, -10.0f, 0f));
```

**Comment by jperedadnr** [comment]
Indeed, there is a bug in this class, and also in `PrismMesh`. There is a rotation [transform](https://github.com/FXyz/FXyz/blob/master/FXyz-Core/src/main/java/org/fxyz3d/shapes/primitives/FrustumMesh.java#L267) applied that fails when the shape axis is parallel to the `Y` axis.

**Comment by jperedadnr** [comment]
Solved with PR #29
